openupgrade_scripts/scripts/hr_contract/16.0.1.0/post-migration.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2023 Coop IT Easy (https://coopiteasy.be)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(env.cr, "hr_contract", "16.0.1.0/noupdate_changes.xml")
```

Comment:
Analyzing noupdate_changes.xml file, the `group_hr_contract_manager` is being modified through `(4, id)`, which means that the previous content is kept, and we don't want this, so you should add first a `(5, )` command for erasing current content.